### ISSUE TITLE
remove warnings for initial checkout

### ISSUE THREAD

**Issue by marc-portier** [vliz-be-opsci/py-RDF-store#1]
when doing a clean checkout and after running

```sh
$ make install
$ make test
``` 

we get a number of warnings that seem to be avoidable

```
=========================================== warnings summary ============================================
../../../../.local/lib/python3.10/site-packages/rdflib/plugins/sparql/__init__.py:52
  /home/mpo/.local/lib/python3.10/site-packages/rdflib/plugins/sparql/__init__.py:52: DeprecationWarning: SelectableGroups dict interface is deprecated. Use select.
    for ep in all_entry_points.get(PLUGIN_ENTRY_POINT, []):

../../../../.local/lib/python3.10/site-packages/rdflib_jsonld/__init__.py:9
  /home/mpo/.local/lib/python3.10/site-packages/rdflib_jsonld/__init__.py:9: DeprecationWarning: The rdflib-jsonld package has been integrated into rdflib as of rdflib==6.0.1.  Please remove rdflib-jsonld from your project's dependencies.
    warnings.warn(

tests/test_store.py: 260 warnings
  /usr/lib/python3.10/random.py:370: DeprecationWarning: non-integer arguments to randrange() have been deprecated since Python 3.10 and will be removed in a subsequent version
    return self.randrange(a, b+1)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

```

**Comment by marc-portier** [comment]
first two warnings are likely due to my own setup - so pls ignore those will cleanup my pip dependencies locally

**Comment by marc-portier** [comment]
closing as 

1. those two latter ones were indeed due to local pip env config (removed pytest, rdflib and rdflib-json from main env and let poetry introduce them locally)
2. and the remaining warning got fixed as part of applying #9 --> the refactoring of the test_store.py cleaned up the non-int argument that was passed into randint()
